chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
I think by the time the validator code is run, the *only* check should be that the authority in the surveillance object isn't null or empty. Based on looking at the rest of the code, the authority property of surveillance is either getting filled in by the UI and passed in on the request or set in the controller or manager code based on the role the user has. I could be missing something but I feel like this can be greatly simplified.